build.sh

```diff
@@ -91,15 +97,14 @@ for WORKER in "${WORKER_DIRS[@]}"; do
   popd
 done
 
-# Compile the schemas into a schema descriptor: compile schemas to proto, compile proto
-# This step will be simplified into a single command in the future
-mkdir -p "${DOWNLOAD_DIR}/generated_protos"
-cp -r "${DOWNLOAD_DIR}"/proto/* "${DOWNLOAD_DIR}/generated_protos"
+# Compile the schemas into a schema descriptor
 mkdir -p "${BUILD_DIR}/SpatialOS/schema/bin"
-
-"${DOWNLOAD_DIR}/schema_compiler" --schema_path="${BUILD_DIR}/SpatialOS/schema" --schema_path="${DOWNLOAD_DIR}" --proto_out="${DOWNLOAD_DIR}/generated_protos" --load_all_schema_on_schema_path "${DOWNLOAD_DIR}"/improbable/*.schema "${BUILD_DIR}"/SpatialOS/schema/*.schema
-"${DOWNLOAD_DIR}/protoc" --proto_path="${DOWNLOAD_DIR}/generated_protos" --descriptor_set_out="${BUILD_DIR}/SpatialOS/schema/bin/schema.descriptor" --include_imports "${DOWNLOAD_DIR}"/generated_protos/*.proto "${DOWNLOAD_DIR}"/generated_protos/**/*.proto
-
-rm -r "${DOWNLOAD_DIR}/generated_protos"
+"${DOWNLOAD_DIR}/schema_compiler" \
+  --schema_path="${BUILD_DIR}/SpatialOS/schema" \
+  --schema_path="${DOWNLOAD_DIR}" \
+  --load_all_schema_on_schema_path \
+  --descriptor_set_out="${BUILD_DIR}/SpatialOS/schema/bin/schema.descriptor" \
+  ${DOWNLOAD_DIR}/improbable/*.schema \
```

Comment:
Just wondering, why is this not in quotes while others are? (I'm not experienced in bash, there might be a reason for this?)

Reply:
Have changed it to make it more consistent.